doc/api/inspector.md

```diff
@@ -0,0 +1,64 @@
+# Inspector
+
+> Stability: 1 - Experimental
+
+The `inspector` module provides an API for interacting with the V8 inspector.
+
+It can be accessed using:
+
+```js
+const inspector = require('inspector');
+```
+
+## Class: inspector.Session
+<!-- YAML
+added: v8.0.0
+-->
+
+The `inspector.Session` class is a primary means of dispatching the messages to
+the V8 inspector backend and receiving message responses and notifications.
+
+At any given moment there may be only one connected session (created either
+with this API or by connecting to an inspector socket). Creating a second
+session using this API will throw an exception.
+
```

Comment:
No. It will detect there's an ongoing session and close the socket, same behaviour as what you get if you are trying to connect from 2 frontends concurrently.